test/golomb-test.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const assert = require('bsert');
+const Block = require('../lib/primitives/block');
+const Opcode = require('../lib/script/opcode');
+const Script = require('../lib/script/script');
+const Golomb = require('../lib/golomb/golomb');
+const testVector = require('./data/filter-valid.json').slice(2);
+
+describe('Golomb test', function() {
+  it('Should match all valid input & output scripts provided as test to their respective filters', function() {
+    const OP_RETURN = Opcode.fromSymbol('return');
+    const L = [];
+
+    // Building an array of arrays to hold the scripts for each block
+    testVector.forEach((testCase, index) => {
+      const [, , blk, prevScripts] = testCase;
+      L[index] = [];
+      const block = Block.fromRaw(Buffer.from(blk, 'hex'));
+      if (prevScripts.length > 0) {
+        const scripts = prevScripts.filter(str => str.length > 0).map(str => Script.fromRaw(str, 'hex'));
```

Comment:
Nit: Use `for ... of` iteration instead of chaining higher order functions. You could iterate over the list once instead of twice by doing so.

Reply:
The line is also too long, there is a max of 80 columns.